### ISSUE TITLE
Safe auto-correction of `Naming/BlockForwarding` and `Style/ArgumentsForwarding` generates a syntax error (Ruby 3.2)

### ISSUE THREAD

**Issue by tagliala** [rubocop/rubocop#12361]
Hi, apologies if this is a duplicate

I've found an error when trying to correct `Naming/BlockForwarding` and `Style/ArgumentsForwarding` at the same time

Example file:

```rb
def some_method(form, **options, &block)
  render 'template', form: form, **options, &block
end
```

Example config:
```yml
AllCops:
  TargetRubyVersion: 3.2
```

--------

## Expected behavior

Generates a valid file

## Actual behavior

```rb
def some_method(form, **, &)
  render(('template', form: form, **, &))
end
```

```
options_and_block.rb:13:21: F: Lint/Syntax: unexpected token tLABEL
(Using Ruby 3.1 parser; configure using TargetRubyVersion parameter, under AllCops)
  render((template, form: form, **, &))
```

## Steps to reproduce the problem

1. Create a file containing like the one in the example

2. Run `bundle exec rubocop options_and_block.rb --only Style/ArgumentsForwarding,Naming/BlockForwarding -a`, targeting

3. Check the output file

## RuboCop version

1.57.2 (and main)

```
bundle exec rubocop -V
1.57.2 (using Parser 3.2.2.4, rubocop-ast 1.30.0, running on ruby 3.2.2) [x86_64-darwin22]
```

```
$ cat .rubocop.yml 
AllCops:
  NewCops: enable
  TargetRubyVersion: 3.2
```

**Comment by tagliala** [comment]
Thanks!
